### ISSUE TITLE
Description alignment

### ISSUE THREAD

**Issue by heldersepu** [swagger-api/swagger-ui#4241]
| Q                               | A
| ------------------------------- | -------
| Bug or feature request?         |  Style Bug
| Which Swagger/OpenAPI version?  | 2.0
| Which Swagger-UI version?       | Latest
| How did you install Swagger-UI? | PetStore
| Which browser & version?        | Chrome & Firefox
| Which operating system?         | Win10

### Demonstration API definition
http://petstore.swagger.io/?defaultModelsExpandDepth=-1&defaultModelExpandDepth=5&filter=store&docExpansion=full#/store/getInventory

### Expected Behavior
All the descriptions should be left aligned 

### Current Behavior
The descriptions are not aligned
![image](https://user-images.githubusercontent.com/545474/36434274-ab5ec676-162c-11e8-8655-9c938c027999.png)

**Comment by erajkovic** [comment]
Also, it would be nice if the vertical alignment did not change when you toggle between "Model" and "Example value" for one operation. Can be reproduced on the same demo link by toggling Model.

**Comment by heldersepu** [comment]
@erajkovic Good point!
The [Try it out] button on the post operation also has a vertical dance going on...

**Comment by shannon7wallace** [comment]
I agree, this would be a great enhancement to the UI! There is very limited real estate given the current behavior.
